documentation/src/docs/transactions.md

```diff
@@ -17,7 +17,7 @@ Samples
 readme](https://github.com/pravega/pravega-samples/blob/master/standalone-examples/README.md).
 
 You really should be familiar with Pravega Concepts (see [Pravega
-Concepts](http://pravega.io/Pravega-Concepts/index.html)) before continuing reading this page.
+Concepts](https://github.com/pravega/pravega/blob/master/documentation/src/docs/pravega-concepts.md)) before continuing reading this page.
```

Comment:
Thanks for reporting and for the contribution, @agarella . I think the right link is this one, though:

http://pravega.io/docs/latest/pravega-concepts/